docs/Developers/Guides/whitelistAddress.md

```diff
@@ -91,14 +131,14 @@ Make sure you are connected to the correct network corresponding to the contract
 
 ### Checking if Your Address Was Successfully Whitelisted/Kissed on Etherscan
 
-
 In the contract that you wan to read, go to the `Read Contract` tab and locate the `tolled` function.
 Enter your wallet address in the `who` (address) field.
 Hit `Query`. The data returned should look like this:
+
 <div style={{textAlign: 'center'}}>
 <img
     src="/img/guides/tolled.png"
     alt="Tolled" 
 />
 </div>
-Once you verify that you are whitelisted, you are ready to read from smart contracts.
\ No newline at end of file
+Once you verify that you are whitelisted, you are ready to read from smart contracts.
```

Comment:
> you are ready to read from smart contracts 

"ready to read from the oracle smart contract" (?)

Reply:
How to check whether whitelisted?

`$ cast call $oracle "tolled(address)(bool)" $my_address --rpc-url $rpc`

Reply:
@pmerkleplant, the second option shows how to verify your whitelisted status using Etherscan
The following command:
`$ cast call $oracle "tolled(address)(bool)" $my_address --rpc-url $rpc` is presented part of Option 1: Using the CLI. I don't think we need to add it again here.